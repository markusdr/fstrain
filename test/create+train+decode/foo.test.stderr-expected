12 / 25 = 0.4800 correct
