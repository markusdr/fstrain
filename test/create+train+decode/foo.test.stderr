6 / 25 = 0.2400 correct
